src/loaders/png/tvgLodePng.cpp

```diff
@@ -2090,17 +2086,7 @@ static unsigned unfilterScanline(unsigned char* recon, const unsigned char* scan
     size_t i;
     switch (filterType) {
         case 0: {
-            if (bytewidth == 4) {
-                for (i = 0; i < length; i += 4) {
-                    //RGBA -> BGRA
-                    recon[i + 0] = scanline[i + 2];
-                    recon[i + 1] = scanline[i + 1];
-                    recon[i + 2] = scanline[i + 0];
-                    recon[i + 3] = scanline[i + 3];
-                }
-            } else {
-                for (i = 0; i != length; ++i) recon[i] = scanline[i];
-            }
+            for (i = 0; i < length; ++i) recon[i] = scanline[i];
```

Comment:
When this change takes effect, https://github.com/Samsung/thorvg/issues/1007#issuecomment-964153604 occurs again. Could you please check?

(https://github.com/Samsung/thorvg/issues/1007#issue-1046668771 Please check the diagram box color on the right of the SVG file attached here)

Reply:
I'll revert this change in this branch, and we can work on the channel order bug in the other branch.

Reply:
Reverted